src/index.js

```diff
@@ -914,6 +914,18 @@ Collection.prototype.all = function () {
   return this.items;
 };
 
+Collection.prototype.concat = function (collection) {
+  if (collection instanceof Collection) {
+    collection = collection.all();
+  }
+
+  for (let iterator = 1; iterator <= collection.count(); iterator++) {
```

Comment:
The problem with forEach in Javascript is that order can be messed up, right? I think keeping order is important.

https://stackoverflow.com/questions/13600922/does-javascript-array-foreach-traverse-elements-in-ascending-order